### PR TITLE
Add support for Philips Hue LWA030

### PR DESCRIPTION
**Pull request by kzaoaai**

### CODE REVIEW

File: src/devices/philips.ts [hunk 1]
```diff
@@ -631,6 +631,13 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Hue white A67 bulb E26 with Bluetooth (1600 Lumen)',
         extend: [philipsLight()],
     },
+    {
+        zigbeeModel: ['LWA030'],
+        model: 'LWA030',
+        vendor: 'Signify Netherlands B.V.',
```
Comment (Koenkk) [inline src/devices/philips.ts:637]:
```suggestion
        vendor: 'Philips',
```

File: src/devices/philips.ts [hunk 2]
```diff
@@ -631,6 +631,13 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Hue white A67 bulb E26 with Bluetooth (1600 Lumen)',
         extend: [philipsLight()],
     },
+    {
+        zigbeeModel: ['LWA030'],
+        model: 'LWA030',
```
Comment (Koenkk) [inline src/devices/philips.ts:636]: What's the model number on the product/site?

Comment (kzaoaai) [reply src/devices/philips.ts:636]: on the box, it says 9290023351B

Comment (kzaoaai) [reply src/devices/philips.ts]: updated

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**